### PR TITLE
Improve cleveref support

### PR DESCRIPTION
**Pull request by torik42**
There are some more symbols which are escaped in the `sed` file. I included all I could find and made it one regEx now. Thus, `\Cref*` now produces `\ref*` which is added to package `hyperref`.

### CODE REVIEW

File: list-of-macros.md [hunk 1]
```diff
@@ -352,6 +352,7 @@ tests: [tests/test\_packages/test\_hyperref.py](tests/test_packages/test_hyperre
 **Macros**
 
 \\href,
+\\ref*,
```
Comment (matze-dd) [inline list-of-macros.md:355]: It is probably better to write `\*`, as '*' has special meaning in Markdown.

**Comment by matze-dd** [comment]
Thank you for these improvements! I've just edited the asterisk escape in the Markdown file and will merge the PR.

File: list-of-macros.md (hunk 1 above)
Comment (torik42) [reply list-of-macros.md]: Okay. This is the same for the `\cref*` commands.

**Comment by torik42** [comment]
Well. I was too slow, but you may also want to escape the asterisk in the `\cref*` commands which are already in `list-of-macros.md`.

**Comment by matze-dd** [comment]
OK, will be done. Thanks again!

**Pull request merged**